src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -106,15 +106,15 @@ BinaryCompactObject::BinaryCompactObject(
     number_of_blocks_++;
   }
 
-  if (object_A_.x_coord >= 0.0) {
+  if (object_A_.x_coord <= 0.0) {
```

Comment:
This change needs to be carried over to some input files (the tests shouldn't pass currently)